packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import Tabs from 'carbon-components-react/es/components/Tabs';
+import Tab from 'carbon-components-react/es/components/Tab';
+import dayjs from 'dayjs';
+import { Order, Encounter, Note, Observation, OrderItem } from '../visit.resource';
+import styles from '../visit-detail-overview.scss';
+import MedicationSummary from './medications-summary.component';
+import NotesSummary from './notes-summary.component';
+import TestsSummary from './tests-summary.component';
+import { OpenmrsResource } from '@openmrs/esm-framework';
+
+function formatTime(date) {
+  return dayjs(date).format('hh:mm');
+}
+
+interface DiagnosisItem {
+  diagnosis: string;
+  order: string;
+}
+
+interface VisitSummaryProps {
+  encounters: Array<Encounter | OpenmrsResource>;
+  patientUuid: string;
+}
+
+const VisitSummary: React.FC<VisitSummaryProps> = ({ encounters, patientUuid }) => {
+  const { t } = useTranslation();
+  const [tabSelected, setSelectedTab] = useState(0);
+
+  const [diagnoses, notes, medications]: [Array<DiagnosisItem>, Array<Note>, Array<OrderItem>] = useMemo(() => {
+    // Medication Tab
+    let medications: Array<OrderItem> = [];
```

Comment:
Hm I made an earlier comment that this should be a `const`. Not sure why this was not implemented as suggested. Please follow my recommendation here. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/51502471/121334181-12291a80-c937-11eb-978c-fb7dec1afa84.png)
I wanted to confirm in here, but no problem, I'll do the changes.

Reply:
What does line 31 have to do with it? My comment was about the inner function.

Reply:
I was saying that this let variable is being assigned to const in the end. I have fixed it now.

Reply:
What you mean is that the value that is referenced by the variable in question here somehow ends up being assigned to another variable, which is then `const`. True, but irrelevant as stated.

Reply:
Got it. will take care of it!